internal/graphql/resolver/meshsync.go

```diff
@@ -4,30 +4,33 @@ import (
 	"context"
 
 	"github.com/layer5io/meshery/internal/graphql/model"
+	"github.com/layer5io/meshkit/broker"
 	"github.com/layer5io/meshkit/database"
 	"github.com/layer5io/meshkit/utils"
 	mesherykube "github.com/layer5io/meshkit/utils/kubernetes"
-	"github.com/layer5io/meshsync/pkg/broker"
 	meshsyncmodel "github.com/layer5io/meshsync/pkg/model"
 )
 
 var (
-	meshsyncName = "meshsync"
-	meshsyncYaml = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_meshsync.yaml"
+	meshsyncName    = "meshsync"
+	meshsyncSubject = "meshery.>"
```

Comment:
Educate me, if you would: why the `.>`?

Reply:
it means any subject that matches meshery.xxx or meshery.xxx.xxx.xx ... and so on. Currently MeshSync publishes to subject name - `meshery.meshsync.discovery`. There could be many more, just namespaced it for meshery in general.